### PR TITLE
Various documentation adjustments

### PR DESCRIPTION
**Pull request by robflop**
**Please describe the changes this PR makes and why it should be merged:**

This PR includes a ton of docs changes, ranging from fixed typos, capitalization ("Discord" instead of "discord", "DataStore" instead of "datastore") and fixed phrasing to expansion of certain parts of the docs.

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/client/Client.js [hunk 1]
```diff
@@ -99,7 +99,8 @@ class Client extends BaseClient {
 
     /**
      * All of the {@link Channel}s that the client is currently handling, mapped by their IDs -
-     * as long as sharding isn't being used, this will be *every* channel in *every* guild, and all DM channels
+     * as long as sharding isn't being used, this will be *every* channel in *every* guild the bot
+	 * is a member of, and all DM channels
```
Comment (iCrawl) [inline src/client/Client.js]: 1 more space pls

File: src/structures/TextChannel.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ class TextChannel extends GuildChannel {
     this.topic = data.topic;
 
     /**
-     * If the Discord considers this channel NSFW
+     * If the guild considers this channel NSFW
```
Comment (KingDGrizzle) [inline src/structures/TextChannel.js]: Wouldn't this make more sense as `If this channel is marked (as?) NSFW`? (idk if the `as` is needed)

**Comment by appellation** [comment]
I don't see the need for mass-pluralizing all the method descriptions.  It seems fine to leave method descriptions as an *action* as opposed to a *result*.  However, I do agree with the capitalization and improved phrasing.

**Comment by robflop** [comment]
The mass-pluralizing was done for two reasons:

 - It seemed inconsistent throughout the docs between pluralized and singular forms
 - I personally think it makes more sense to have them pluralized

File: src/structures/TextChannel.js (hunk 2 above)
Comment (iCrawl) [reply src/structures/TextChannel.js]: no

File: src/structures/User.js [hunk 3]
```diff
@@ -134,7 +134,7 @@ class User extends Base {
   }
 
   /**
-   * The Discord "tag" for this user
+   * The Discord "tag" (e.g. `hydrabolt#0086`) for this user
```
Comment (hydrabolt) [inline src/structures/User.js]: 😆

Comment (iCrawl) [reply src/structures/User.js]: 👎

**Pull request merged**